prime-router/metadata/fhir_mapping/hl7/datatype/Quantity.yml

```diff
@@ -1,5 +1,11 @@
 value_1 :
   condition: $value CONTAINS_STRING '.'
+  #  Not specifying a type results in the full "value" being reported correctly and no rounding/float conversion occurs.
+  #  However, it also breaks the integration tests, so we are going to have to duplicate this value into an extension
```

Comment:
How does it break the integration tests? Why can't we just change the integration tests?

Reply:
@JessicaWNava Agreed, it is an intrusive workaround that may have unintended side effects and we're Ok with finding another solution for what we need. We have an open PR in the `LinuxForHealth/hl7v2-fhir-converter` repo for a permanent fix, but in the meantime I think we're Ok with leaving it as is